### PR TITLE
consistent error handling (mu-error)

### PR DESCRIPTION
**Pull request by davidmarkclements**
cc @mcollina

### CODE REVIEW

**Comment by davidmarkclements** [comment]
closes #22

**Comment by davidmarkclements** [comment]
forgot mu-error dep - updated

File: core/router.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
 var bloomrun = require('bloomrun')
 var assert = require('assert')
 var stringify = require('fast-safe-stringify')
-var errors = require('./err')
+var mue = require('mu-error')({dev: process.NODE_ENV !== 'production'})
```
Comment (mcollina) [inline core/router.js]: I don't think this should happen here, but it should be a configuration of `mu` itself

**Comment by davidmarkclements** [comment]
hah - yes agreed - I just did that before I looked at this. 

That allows for `mu.error` to be used by users of mu

File: core/router.js (hunk 1 above)
Comment (davidmarkclements) [inline core/router.js]: hah - yes agreed - I just did that before I looked at this. 

That allows for `mu.error` to be used by users of mu

**Pull request merged**